src/cljs/main/broadfcui/page/workspace/monitor/common.cljs

```diff
@@ -41,25 +41,29 @@
 
 (def success-icon
   (icons/icon {:style {:color (:success-state style/colors) :marginRight 4
-                       :width table-style/table-icon-size :height table-style/table-icon-size}}
+                       :width table-style/table-icon-size :height table-style/table-icon-size}
+               :data-test-id "status-icon" :data-test-value "success"}
```

Comment:
I can not put `(config/when-debug ...)` around these values because, for some reason unknown to me, the config is consistently not yet loaded when these are rendered. Putnam suggested adding a watch to the config atom but I don't see how that's going to work in this context. I'm going to need some additional help fixing this. It would be great to fix this before merging but I'm not going to hold up the merge for this one case.